pravega/pravega-operator
# Issue 441: Name of the configmap in tools/manifest should be changed to supported-versions-map

Signed-off-by: prabhaker24 <prabhaker.saxena@dell.com>

### Change log description
Name of the configmap in tools/manifest should be changed to supported-versions-map

### Purpose of the change
FIxes #441 

### What the code does
Name of the configmap in tools/manifest should be changed to supported-versions-map which would make it consistent with the configmap name installed by helm charts for versions

### How to verify it
upgrade pravega operator for 0.4.3 to 0.5.1 and after than update the config map and have 0.8.0 in keys and try and upgrade the pravega operator should trigger the upgrade.

_(Steps to verify that the changes are effective)_
